packages/-ember-data/tests/integration/identifiers/configuration-test.ts

```diff
@@ -148,9 +151,19 @@ module('Integration | Identifiers - configuration', function (hooks) {
     let updateMethodCalls = 0 as number;
     let updateCallback: (...args: any[]) => void;
 
-    function updateMethod(identifier: StableRecordIdentifier, data: ExistingResourceObject, bucket: string) {
-      updateMethodCalls++;
-      updateCallback!(identifier, data);
+    function updateMethod(
+      identifier: StableIdentifier | StableRecordIdentifier,
+      data: ResourceData | unknown,
+      bucket: IdentifierBucket
+    ) {
+      switch (bucket) {
+        case 'record':
+          updateMethodCalls++;
+          updateCallback!(identifier, data);
```

Comment:
the unsafe cast isn't new here (though it seems it due to the indentation switch) but the TL;DR is it is defined in the the individual test setup, but we define it with `let` because this is is shared setup for each test.

Reply:
Ah, got it. Thanks for the clarification.